test/integration/auth.spec.ts

```diff
@@ -1758,6 +1758,46 @@ describe('admin.auth', () => {
       });
     });
 
+    describe('Passkey config management operations', () => {
```

Comment:
can we also add a disable test case?

Reply:
Disabling is not supported by the backend as of now.

Reply:
confirmed with backend disabling is an undone feature.  Resolving this.